app/jobs/weekly_email.rb

```diff
@@ -0,0 +1,13 @@
+class WeeklyEmail < ApplicationJob
+  #sidekiq_options retry: false
+   queue_as :default
+   def perform(*)
+    #@users = User.all
+    #@users.each do |user|
+     # mail = ApplicationMailer.sample_email(id: user.id, email: user.email)
+      #mail.deliver_now
+
+    #end
+    puts "Hola desde weeklymailer"
```

Comment:
are these comments required? btw, try to avoid sending unnecessary puts in your PRs